### DISCUSSION TITLE
Avoid duplicates pages when using localized pathnames

### DISCUSSION THREAD

**Discussion by chuckcg** [amannn/next-intl#671] [Q&A] [answered]
At the moment, if I have this in my pathnames:
```
export const pathnames = {
  '/': '/',
  '/about': {
    en: '/about-us',
    fr: '/a-propos',
    nl: '/over-ons',

  }
}
```

Then both **domain.com/en/about** and **domain.com/en/about-us** will work and show the same page. Is there a way to make the /en/about page 301 redirect to /en/about-us ?

**Comment by navandstokes** [comment, marked as answer]
Change to 

```
export const pathnames = {
  '/': '/',
  '/about-us': {
    en: '/about-us',
    fr: '/a-propos',
    nl: '/over-ons',

  }
}
```

And your file structure to `/about-us/page.tsx`

**Reply by chuckcg** [reply]
Ah yeah, makes sense 🤦 thanks
